bears/vcs/git/GitCommitBear.py

```diff
@@ -6,6 +6,13 @@
 from coalib.results.Result import Result
 
 
+def none_or_bool(value):
+    if value is None:
+        return None
+    else:
+        return bool(value)
```

Comment:
this function is useless, a setting (and value is a setting) will never be the None object